Script.js

```diff
@@ -0,0 +1,52 @@
+const startButton = document.querySelector('#startGameButton');
```

Comment:
Suggest renaming this whole file to `script.js` with a lowercase s to match the script tag in your HTML